Chapter-4-applying-tactical-domain-driven-design/Fitnet.Common/Fitnet.Common.Core/ValueObject.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Common.Core;
+
+public abstract class ValueObject
```

Comment:
```suggestion
public abstract class ValueObject : IEquatable<ValueObject>
```

Can we implement interface IEquatable that will ensure that ValueObject behaves correctly and consistently with collections. 

What is your opinion about it?

Reply:
We could as well add unit tests that test ensure value objects equalability works during comparision in collection for e.g in LINQ Where statment.

Reply:
Great spot! I completely forgot about it :)